ui/imports/Utils.qml

```diff
@@ -50,12 +50,18 @@ QtObject {
     }
 
     function getMessageWithStyle(msg, useCompactMode, isCurrentUser) {
+        let fMsg = msg.startsWith('<code>') ? `<div class='code-wrapper'>${msg}</div>` : msg
+        const codeBg = Style.current.name === 'dark' ? Style.current.darkGrey : Style.current.lightGrey
```

Comment:
We can just add a new Style property for `codeBg`, that way you don't need the ternary here.